app/controllers/register_controller.rb

```diff
@@ -5,13 +5,16 @@ def new
 
   def create
     @user = User.new(user_params)
+    @user.role = 1
```

Comment:
By default visitors have a role of 1, so when we're creating a new user - we have to assign the role to 1